litmus-portal/graphql-server/pkg/chaos-workflow/handler/handler.go

```diff
@@ -81,6 +81,10 @@ func DeleteChaosWorkflow(ctx context.Context, projectID string, workflowID *stri
 		return false, err
 	}
 
+	if workflow.IsRemoved {
```

Comment:
Instead of this, you may apply filter in the query itself: {"is_removed": true}, and add a check for that.